experimenter/experimenter/nimbus_ui_new/static/css/style.scss

```diff
@@ -41,6 +41,15 @@
     calc(0.75em + 0.375rem) calc(0.75em + 0.375rem);
 }
 
+.bootstrap-select.is-invalid {
```

Comment:
🙃

Reply:
I know it's a hassle, bootstrap-select doesn't seem to style itself consistently with the other bootstrap5 elements so I gotta keep overriding it.